components/suggestion/search_date_suggestion/search_date_suggestion.tsx

```diff
@@ -13,10 +13,14 @@ import 'react-day-picker/lib/style.css';
 
 import 'moment';
 
-const loadedLocales = {};
+const loadedLocales: any = {};
```

Comment:
I think this should be something like `Record<string, moment.Locale>` instead of `any`

Reply:
Thank you. Changing added.